birdhouse/read-configs.include.sh

```diff
@@ -126,6 +126,14 @@ source_conf_files() {
           # fix immediately.
           # The new adir with typo will not be active but at least all the existing
           # will still work.
+          #
+          # Do not exit on not existing conf dir also allow for smooth
```

Comment:
Suggested:

```
# Not exiting also allows for a smooth transition when component paths change between updates.
```

Reply:
This one I decided to keep the specific part about `not existing conf dir` because it is code comment so should be more precise.  New wording: "Allowing not existing conf dir also helps for smooth transition of component path when they are new/renamed/deleted."